### ISSUE TITLE
Would add Prometheus metrics for observability?

### ISSUE THREAD

**Issue by liukch** [thedevs-network/kutt#551]
It is important for self-hosted server to have good observability
look for this https://grafana.com/oss/prometheus/exporters/nodejs-exporter/ to get more information about Node.js Exporter

**Comment by nlamirault** [comment]
any news on this feature? We haven't got monitoring / alerting on Kutt.

**Comment by liukch** [comment]
@nlamirault No, Kutt has been running very stable in the production environment.

**Comment by Tchoupinax** [comment]
Hello,

A merge request is opened [here](https://github.com/thedevs-network/kutt/pull/696). Adding several types of metrics is WIP.

**Comment by Avariya** [comment]
> [@nlamirault](https://github.com/nlamirault) No, Kutt has been running very stable in the production environment.

Thanks for your input, @nlamirault. While Kutt has generally been stable, we experienced a DDoS attack on our instance recently, during which it went down for over 30 minutes under a load of just 2k RPS.

Having Prometheus metrics would help us significantly - especially for monitoring and configuring autoscaling to better handle such incidents in the future. Metrics would give us more visibility into bottlenecks and resource usage, enabling more proactive scaling and quicker response times during traffic spikes.
